registrations/migrations/0024_signup_and_signup_group_protected_data.py

```diff
@@ -0,0 +1,156 @@
+                registration=signup.registration_id,
+                signup=signup.id,
+                extra_info=signup.extra_info,
+                date_of_birth=signup.date_of_birth,
+            )
+        )
+    SignUpProtectedData.objects.bulk_create(signup_protected_datas)
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("registrations", "0023_rename_timestamp_fields"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="SignUpProtectedData",
+            fields=[
+                (
+                    "id",
+                    models.AutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                (
+                    "extra_info",
+                    encrypted_fields.fields.EncryptedTextField(
+                        blank=True, default=None, null=True, verbose_name="Extra info"
+                    ),
+                ),
+                (
+                    "registration",
+                    models.ForeignKey(
+                        on_delete=deletion.PROTECT,
+                        related_name="signupprotecteddatas",
+                        to="registrations.registration",
+                    ),
+                ),
+                (
+                    "date_of_birth",
+                    encrypted_fields.fields.EncryptedDateField(
+                        blank=True, null=True, verbose_name="Date of birth"
+                    ),
+                ),
+                (
+                    "signup",
+                    models.OneToOneField(
+                        default=None,
+                        null=True,
+                        on_delete=deletion.SET_NULL,
+                        related_name="protected_data",
+                        to="registrations.signup",
+                    ),
+                ),
+            ],
+            options={
+                "abstract": False,
+            },
+        ),
+        migrations.CreateModel(
+            name="SignUpGroupProtectedData",
+            fields=[
+                (
+                    "id",
+                    models.AutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                (
+                    "extra_info",
+                    encrypted_fields.fields.EncryptedTextField(
+                        blank=True, default=None, null=True, verbose_name="Extra info"
+                    ),
+                ),
+                (
+                    "registration",
+                    models.ForeignKey(
+                        on_delete=deletion.PROTECT,
+                        related_name="signupgroupprotecteddatas",
```

Comment:
Not really an issue, but data is the plural form of datum i.e. there's no such thing as datas. :wink: